routes/usersRoutes.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const pool = require("../db/db_connect");
+
+const router = express.Router();
+router.use(bodyParser.json());
+
+//Getting all users
+//NB: Only for administrators
+router.get("/", async (req, res) => {});
+
+//Getting user information
+router.get("/:userId", async (req, res) => {});
+
+//creating user
+router.post("/", async (req, res) => {});
+
+//Patch username
+router.patch("/:userId/username", async (req, res) => {});
+
+//Delete password
+router.delete("/:userId/password", (req, res) => {});
```

Comment:
Can someone delete a password ?

Reply:
Ohh, sorry I would like to say when you want to reset your password.

Reply:
> Ohh, sorry I would like to say when you want to reset your password.

okay, You can make it `POST /reset`.

Reply:
Okay